daemon/cluster/cluster.go

```diff
@@ -509,6 +509,14 @@ func (c *Cluster) stopNode() error {
 	return nil
 }
 
+func atLeastOneMoreNodeReachable(reachable, unreachable int) bool {
```

Comment:
hum true, I should change it to a more *domain* name :angel: 
Updating it :wink: